frame/nfts/src/lib.rs

```diff
@@ -838,7 +841,7 @@ pub mod pallet {
 					Error::<T, I>::NoPermission
 				);
 			}
-			Self::do_mint(collection, item, owner, item_config, true, |_, _| Ok(()))
+			Self::do_mint(collection, item, owner.clone(), owner, item_config, true, |_, _| Ok(()))
```

Comment:
q: why are we passing `owner.clone()` as a caller and not using `let caller = ensure_signed(origin)?;` like we do in `mint`?

Reply:
The `force_mint()` method supports root calls, that's why we can't simply put `ensure_signed(origin)`. On the other hand, `Self::do_mint` needs to know a caller to take the deposit.
Idk, maybe we should rename `caller` into `depositor` in the `do_mint` method, wdyt?

Reply:
> Idk, maybe we should rename caller into depositor in the do_mint method, wdyt?

I'd be in favor

Reply:
I think `owner` is confusing since `CollectionRole` has an `Owner` variant. What about in `do_mint()`, `force_mint()`, and `mint()` we use:

- `beneficiary`: Account into which the item will be minted. (Consistent with [assets](https://github.com/paritytech/substrate/blob/8e76398/frame/assets/src/lib.rs#L771))
- `depositor`: Account placing the deposit.

Then the `owner` term is specific to the collection role.

Reply:
done